learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -248,63 +267,67 @@ Accessing Scene Nodes
 ~~~~~~~~~~~~~~~~~~~~~
 
 One of the most common tasks is accessing Scene Tree Nodes (again, not to mistake with *Visual Script Nodes*).
-Dragging from the Scene Tree and dropping into the canvas, by default, will ask you to *call a method* (sometimes referred to as *member function*) on
-this node. 
+Dragging from the Scene Tree and dropping into the canvas, by default, will ask you to *call a method* (sometimes referred to as *member function*) on this node. 
 
 .. image:: img/visual_script19.png
 
+
 While accessing properties is desired in most cases (more on that below), sometimes *calling methods* can be useful too.
 Methods execute specific actions on objects. In the above case, the mouse pointer can be warped to a position in local
 coordinates to the control. Another common use case is queueing a node for deletion, which is done with the *queue_free* method.
 
 .. image:: img/visual_script20.png
 
+
 Care must be taken that this only works if the scene being edited contains your *Visual Script* in one of the nodes! Otherwise, a warning will be shown.
 
+
 Accessing Scene Node Properties
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This will be by far the most common way to *Scene Nodes* in Visual Scripting. Simply select a *Scene Node* from the *Scene Tree*,
-go the the inspector, find *The Name* of the property you want to edit (hint, *not* the value!) and drag it to the canvas:
+This will be by far the most common way to edit *Scene Nodes* in Visual Scripting. Simply select a *Scene Node* from the *Scene Tree*, go to the inspector, find *The Name* of the property you want to edit (hint, *not* the value!) and drag it to the canvas:
```

Comment:
"This is the most common way"
Never use "Simply"
Capitalize "Inspector"
"*The Name*" -> "the *Name*"